server/routes/wishCards.js

```diff
@@ -199,6 +199,82 @@ router.post(
   },
 );
 
+// @desc    Edit Wishcard
+// @route   PUT '/wishcards/edit/wishcardId'
+// @access  Private, verified partner or admin
+// @tested 	Yes
+router.post(
+  '/edit/:id',
+  limiter,
+  renderPermissions,
+  createWishcardValidationRules(),
+  validate,
+  async (req, res) => {
+    try {
+      let { childBirthday, wishItemPrice } = req.body;
+      const wishcard = await WishCardRepository.getWishCardByObjectId(req.params.id);
+
+      childBirthday = childBirthday ? new Date(childBirthday) : wishcard.childBirthday;
+      wishItemPrice = wishItemPrice ? Number(wishItemPrice) : wishcard.wishItemPrice;
+
+      await WishCardRepository.updateWishCard(wishcard._id, {
+        childBirthday,
+        wishItemPrice,
```

Comment:
you can set the bithday and wishitemprice inside the parameters like you did with the address